packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,68 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the post_install step in expo/scripts/autolinking.rb
+
+- (NSURL *)devLauncherURL
+{
+#if RCT_DEV
```

Comment:
Im not sure of the implications of using  `RCT_DEV` - I think it should be fine but I think we've used `DEBUG` other places? If you know the differences I would appreciate your input on it!

Reply:
In reviewing this, it seems like we don't need it. `EX_DEV_LAUNCHER_URL` is only defined for debug builds anyway. So I reverted this change.